Source/missiles.cpp

```diff
@@ -3724,39 +3688,83 @@ void ProcessRhino(Missile &missile)
 	PutMissile(missile);
 }
 
-void ProcessFireWallControl(Missile &missile)
+/*
+ * @brief Moves the control missile towards the right and grows walls.
+ * @param missile The control missile.
+ * @param sourceIdx The source index of the wall missile.

```

Comment:
Do we need to include this as parameter? We could get it from `Missile` parameter or?